lib/matplotlib/legend.py

```diff
@@ -348,7 +348,7 @@ def __init__(self, parent, handles, labels,
                  bbox_to_anchor=None,  # bbox that the legend will be anchored.
                  bbox_transform=None,  # transform for the bbox
                  frameon=None,  # draw frame
-                 handler_map=None,
+                 handler_map=None
```

Comment:
would leave the comma (per https://stackoverflow.com/questions/11597901/why-are-trailing-commas-allowed-in-a-list)